spec/additional-sample-data/valid/gazette-notice_fr-bodacc-bilan-01.json

```diff
@@ -0,0 +1,47 @@
+{
+    "issue": {
+        "publication": {
+            "publisher": {
+                "name": "Direction de l'information légale et administrative",
+                "url": "http://www.dila.premier-ministre.gouv.fr/"
+            },
+            "jurisdiction_code": "fr",
+            "title": "Bulletin officiel des annonces civiles et commerciales",
+            "url": "http://www.bodacc.fr/"
+        },
+        "identifier": "20150001",
+        "edition_id": "C",
+        "url": "ftp://echanges.dila.gouv.fr/BODACC/2015/BILAN_BXC20150001.taz"
+    },
+    "date_published": "2015-01-05",
+    "source_url": "ftp://echanges.dila.gouv.fr/BODACC/2015/BILAN_BXC20150001.taz",
+    "sample_date": "2016-01-06T16:05:46Z",
```

Comment:
I'm using `retrieved_at` since we seemed to agree that there were no consistent semantics for this field, and that `retrieved_at` would be fine.